lib/Conversion/SCFToCalyx/SCFToCalyx.cpp

```diff
@@ -1103,36 +1094,38 @@ struct FuncOpConversion : public calyx::FuncOpPartialLoweringPattern {
     auto *compState = loweringState().getState<ComponentLoweringState>(compOp);
     compState->setFuncOpResultMapping(funcOpResultMapping);
 
+    unsigned extMemCounter = 0;
```

Comment:
Can you write a unit CHECK test for this please?

Reply:
Can you elaborate a bit more on this?
Like when we have a mix of different types of function arguments, out of which some are `memrefs` and some are not? An how can I check an `unsigned` variable here?

Reply:
I meant to say you've added changes to the SCF to Calyx pass, but is there a test that confirms the behavior of what you're trying to do?

Reply:
No I haven't, will push soon!
I might have asked this, but I've been wondering if there is any tool to generate `CHECK: ...`/`CHECK-LABEL:...`?